apps/web/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
```

Comment:
If the svg tests aren't working on web, let's get rid of the test on web and delete this file + the new svg-inline-loader dep